package-res/resources/web/dataapi/models-svc.js

```diff
@@ -55,7 +55,7 @@ pentaho.pda.SvcHandler.prototype.getSources = function(callback, options) {
             var contextName = options['context'];
             var query = "domainName=&"+((contextName)? 'context='+contextName : "");
             var jsonStr = pentahoPost( url, query, null, 'text/text' );
```

Comment:
I would prefer that we move away from pentahoPost and to the dojo request method. you can specify the return type and not have to parse it after the fact.